easybuild/easyconfigs/g/GenomeWorks/GenomeWorks-2021.02.2-fosscuda-2020b.eb

```diff
@@ -0,0 +1,135 @@
+# author: Denis Kristak (INUITS)
+easyblock = 'CMakePythonPackage'
+
+name = 'GenomeWorks'
+version = '2021.02.2'
+local_git_commit = '691380d1122a9ad86f91b3038c8b6625d137b6ed'
+
+homepage = 'https://github.com/clara-parabricks/GenomeWorks'
+description = "SDK for GPU accelerated genome assembly and analysis"
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/clara-parabricks/GenomeWorks/archive/'],
```

Comment:
Wouldn't it be *a lot* easier to download via `git_config` here, with `recursive` enabled?

See https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-git-config

We recently switched to this for `PyTorch` (see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.1-foss-2020b.eb#L10), and it simplifies the easyconfig file quite a bit (and it's way easier to maintain).

Reply:
This was my idea, since I did similar thing in Racon https://github.com/easybuilders/easybuild-easyconfigs/pull/10258/files
Does the `git_config` + `recursive` keeps things reproducible? I remember that there was the issue with the checksums (downloading + making a zip again)

Reply:
We can't include a checksum, but it is reproducible, since we're cloning a specific git commit.

Reply:
Switch to `git_config` done in 142f9f7, which leads to a much cleaner easyconfig...